OrbitQt/TopDownWidget.cpp

```diff
@@ -374,3 +396,41 @@ QVariant TopDownWidget::HookedIdentityProxyModel::data(const QModelIndex& index,
       QStringLiteral("] ");
   return kDisplayHookedPrefix + data.toString();
 }
+
+void TopDownWidget::ProgressBarItemDelegate::paint(QPainter* painter,
```

Comment:
Nit: Add a comment that this is not really a progress but that we are using this to visualize the percentage in the cell.

Reply:
Done.